### ISSUE TITLE
[Feature Request] add option to show all hidden lines

### ISSUE THREAD

**Issue by ctf0** [enlightn/enlightn#13]
**Is your feature request related to a problem? Please describe.**
its a bit frustrating when i cant see the hidden lines

**Describe the solution you'd like**
the max visible lines are 5 & so having away to get around that would be very helpful

**Describe alternatives you've considered**
`php artisan enlightn | more`, did nothing

**Additional context**
![Screen Shot 2021-01-25 at 9 57 41 AM](https://user-images.githubusercontent.com/7388088/105676974-f545fa80-5ef3-11eb-8334-1664de6ddea9.png)

**Comment by paras-malhotra** [comment]
Hi @ctf0, thanks for posting the feature request. We'll consider it for the next release.

In the meantime, you can run individual analyzers using the arguments to display all the lines:

```bash
php artisan enlightn Enlightn\\Enlightn\\Analyzers\\Reliability\\DeadCodeAnalyzer
```

You can provide multiple arguments as well:

```bash
php artisan enlightn Enlightn\\Enlightn\\Analyzers\\Reliability\\DeadCodeAnalyzer Enlightn\\Enlightn\\Analyzers\\Security\\CSRFAnalyzer
```

**Comment by ctf0** [comment]
i was going to actually ask to add the analyzer class names to the docs or somewhere instead of having to search the codebase, thanx for the quick reply.

**Comment by paras-malhotra** [comment]
@ctf0 yes, that seems to be a nice suggestion. We'll make sure to add the class names to the docs 👍

**Comment by paras-malhotra** [comment]
@ctf0 this is now available in v1.5. Also, the docs now include the class names 👍

**Comment by ctf0** [comment]
fockin awesome, many thanks
